save/new_account_iam_role.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python3
```

Comment:
This `save/` directory seems like it doesn't belong. Accidentally included in a commit?

Reply:
ok